components/rpill/common/rpill.cc

```diff
@@ -33,8 +33,14 @@ bool IsUncertainFuture(DeviceInfo* device_info_ptr) {
 
   const std::string device_id = device_info_ptr->id();
```

Comment:
should either lowercase this string or leave a comment that explains why we can assume it is lowercased.

Reply:
`id()` function converts to lowercase